packages/mdc-textfield/foundation.ts

```diff
@@ -503,21 +455,20 @@ class MDCTextFieldFoundation extends MDCFoundation {
   }
 
   /**
-   * @return {!Element|!NativeInputType} The native text input from the
-   * host environment, or a dummy if none exists.
-   * @private
+   * host environment, or a dummy if none exists
```

Comment:
Lost the first line of this parameter description too

Reply:
Done